doc/release-notes/8868-fix-json-import.md

```diff
@@ -0,0 +1,5 @@
+The dataset export has been updated to eliminate the export of "undefined" as a metadata language.
+
+Run ReExportall to update Exports
+
+Following the directions in the [Admin Guide](http://guides.dataverse.org/en/5.12/admin/metadataexport.html#batch-exports-through-the-api)
```

Comment:
@landreev, I added a quick release note. Please feel free to edit as you wish.

Reply:
Perfect.

Reply:
Going to take you up on that "feel free" - adding a mention of the duplicate keyword in the DDI under bugfixes in the note. (I want to merge this PR tonight!)